lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -158,6 +165,56 @@ sub view : Private {
     );
 }
 
+sub _files_to_categories {
+    my ( $release, $files ) = @_;
+    my $ret = +{
+        provides          => [],
+        documentation     => [],
+        documentation_raw => [],
+        modules           => [],
+    };
+
+    for my $f ( @{$files} ) {
+        if ( length $f->{module} ) {
+            push @{ $ret->{modules} } => $f if $f->{documentation};
```

Comment:
Any particular reason for the fat comma here?  I think a plain old comma might be easier to read.